sdk/rust/oak/src/http/util.rs

```diff
@@ -89,6 +94,30 @@ impl Pipe {
     }
 }
 
+impl Into<HttpPipe> for Pipe {
```

Comment:
I must say I am getting confused with all these pipe-like structs :) I guess they are necessary, but we definitely need how to unify or make them more understandable at some point.